src/main/java/com/projecturanus/betterp2p/client/gui/widget/WidgetP2PDevice.kt

```diff
@@ -82,6 +87,10 @@ class WidgetP2PDevice(private val selectedInfoProperty: KMutableProperty0<InfoWr
     }
 
     private fun drawButtons(gui: GuiScreen, info: InfoWrapper, mouseX: Int, mouseY: Int, partialTicks: Float) {
+        info.renameButton.xPosition = x
```

Comment:
why the rename button is the whole widget

Reply:
eeeeeeeeeee

Reply:
![Y4P _Z694VZ37848)F FWYX](https://user-images.githubusercontent.com/33802162/189523402-be3cca26-ef09-453f-b3c9-f3e8e87e94d6.gif)